nixos/modules/services/networking/coturn.nix

```diff
@@ -341,25 +341,66 @@ in {
           '' }
           chmod 640 ${runConfig}
         '';
-        serviceConfig = {
+        serviceConfig = rec {
```

Comment:
Thanks, I created that list for synapse based on https://bgpfilterguide.nlnog.net/guides/bogon_prefixes/. Was thinking about including it, but that would make it hard to override, unless we make it an option.

That being said, coturn supports a `denied-peer-ip` setting, but it wants a non-cidr `<from>-<to>` format from what I remember.

```nix
    extraConfig = ''
      denied-peer-ip=10.0.0.0-10.255.255.255
      denied-peer-ip=192.168.0.0-192.168.255.255
      denied-peer-ip=172.16.0.0-172.31.255.255
    '';
```

Reply:
`IPAddressAllow` always takes precedence over `IPAddressDeny`, so overriding would be somewhat trivial. That said I'm very much not sure about how this would affect deployments

Reply:
I generally try not to make people override the systemd unit, as that is not well disoverable for new users.